### PR TITLE
Add proofreading changes from Dublin to SLE Storage Guide

### PR DESCRIPTION
**Pull request by lproven**
### Description
Incorporate Dublin's requested changes to the Storage Guide.

2 important notes:
• I found additional uses of "commend" in place of "command" and fixed them. I also fixed some missing punctuation.
• I reformatted the storage_multipath.xml file as the formatting was strange. I also fixed a set of broken ranges with mismatched parentheses: (1-10], (10-20] etc. I have also found this typo in the Debian and Ubuntu docs; I think nobody dared to question it.

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLE15SP1
- [ ] To maintenance/SLE15SP0
- [ ] To maintenance/SLE12SP5
- [ ] To maintenance/SLE12SP4
- [ ] To maintenance/SLE12SP3

### CODE REVIEW

File: xml/common_intro_lifecycle_support_i.xml [hunk 1]
```diff
@@ -124,7 +124,7 @@
   <itemizedlist>
    <listitem>
     <para>
-     Technology previews
+     Technology previews.
```
Comment (cjschroder) [inline xml/common_intro_lifecycle_support_i.xml:127]: According to the style guide, this should be 'technology previews' with no caps or period
https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-lists

File: xml/storage_iscsi.xml [hunk 2]
```diff
@@ -1298,13 +1298,13 @@ o- / ............................ [...]
...
-    To familiarize yourself with functionality of targetcli, set up a local
+    To familiarize yourself with the functionality of targetcli, set up a local
     image file as a software target using the <command>create</command> command:
    </para>
 <screen>/backstores/fileio create test-disc <replaceable>/alt</replaceable>/test.img 1G</screen>
    <para>
-    This creates the 1GB <filename>test.img</filename> image in the specified
-    location (in this case <filename>/alt</filename>). Run
+    This creates a 1&nbsp;GB <filename>test.img</filename> image in the
+    specified location (in this case <filename>/alt</filename>). Run
```
Comment (cjschroder) [inline xml/storage_iscsi.xml:1307]: there should be a space between 1 and GB

File: xml/storage_iscsi.xml [hunk 3]
```diff
@@ -1434,7 +1434,7 @@ o- tpg1 .............................................. [no-gen-acls, no-auth]
       o- portals ............................................... [Portals: 1]
         o- 0.0.0.0:3260 ................................................ [OK]</screen>
    <para>
-    There is now an iSCSI target that has a 1GB file-based back-store. The
+    There is now an iSCSI target that has a 1&nbsp;GB file-based backstore. The
```
Comment (cjschroder) [inline xml/storage_iscsi.xml:1437]: needs a space between 1 and GB

File: xml/storage_multipath.xml [hunk 4]
```diff
@@ -63,77 +63,7 @@
...
...member>IBM 1722-600</member><member>IBM 1724</member><member>IBM 1726</member><member>IBM 1742</member><member>IBM 1745, 1746</member><member>IBM 1750500</member><member>IBM 1814</member><member>IBM 1815</member><member>IBM 1818</member><member>IBM 1820N00</member><member>IBM 2105800</member><member>IBM 2105F20</member><member>IBM 2107900</member><member>IBM 2145</member><member>IBM 2810XIV</member><member>IBM 3303 NVDISK</member><member>IBM 3526</member><member>IBM 3542</member><member>IBM IPR</member><member>IBM Nseries</member><member>IBM ProFibre 4000R</member><member>IBM S/390 DASD ECKD</member><member>IBM S/390 DASD FBA</member><member>Intel Multi-Flex</member><member>LSI/ENGENIO INF-01-00</member><member>NEC DISK ARRAY</member><member>NETAPP LUN</member><member>NEXENTA COMSTAR</member><member>Pillar Axiom</member><member>PIVOT3 RAIGE VOLUME</member><member>SGI IS</member><member>SGI TP9100, TP 9300</member><member>SGI TP9400, TP9500</member><member>STK FLEXLINE 380</member><member>STK OPENstorage D280</member><member>SUN CSM200_R</member><member>SUN LCSM100_[IEFS]</member><member>SUN STK6580, STK6780</member><member>SUN StorEdge 3510, T4</member><member>SUN SUN_6180</member>
```
Comment (cjschroder) [inline xml/storage_multipath.xml:66]: Is there a reason for this to be a blob instead of nicely-formatted with line breaks between each set of 'member' tags?

File: xml/storage_multipath.xml [hunk 5]
```diff
@@ -179,13 +109,7 @@
...
     &productname;:
    </para>
-   <simplelist>
-    <member>EMC</member>
-    <member>Hitachi</member>
-    <member>Hewlett-Packard/Compaq</member>
-    <member>IBM</member>
-    <member>NetApp</member>
-    <member>SGI</member>
+   <simplelist><member>EMC</member><member>Hitachi</member><member>Hewlett-Packard/Compaq</member><member>IBM</member><member>NetApp</member><member>SGI</member>
```
Comment (cjschroder) [inline xml/storage_multipath.xml:112]: again, this seems wrong, should have line breaks between 'member' tags

File: xml/storage_multipath.xml [hunk 6]
```diff
@@ -364,7 +288,8 @@
     that the <filename>initrd</filename> and the installed system behave the
     same regarding the root file system and all other file systems required to
     boot the system. If Multipath is enabled in the system, it also needs to be
-    enabled in the <filename>initrd</filename> and vice versa. See <xref
+    enabled in the <filename>initrd</filename> and vice versa. See
```
Comment (cjschroder) [inline xml/storage_multipath.xml:291]: Seems like this should say `enabled in the <filename>initrd</filename> file`

File: xml/storage_multipath.xml [hunk 7]
```diff
@@ -1807,7 +1733,8 @@ blacklist_exceptions {
     Starting with &productname; 12 SP2, the multipath tools support the option
     <literal>find_multipaths</literal> in the <literal>defaults</literal>
     section of <filename>/etc/multipath.conf</filename>. Simply put, this
-    option prevents multipath and <systemitem
+    option prevents multipath and
```
Comment (cjschroder) [inline xml/storage_multipath.xml:1736]: maybe getting a little picky here, but 'simply put' is unnecessary

File: xml/storage_multipath.xml [hunk 8]
```diff
@@ -3908,11 +3833,7 @@ umount /mnt</screen>
     <para>
      For example:
     </para>
-    <simplelist>
-     <member><literal>Number Major Minor RaidDevice State</literal></member>
-     <member><literal>0 253 0 0 active sync /dev/dm-0</literal></member>
-     <member><literal>1 253 1 1 active sync /dev/dm-1</literal></member>
-     <member><literal>2 253 2 2 active sync /dev/dm-2</literal></member>
+    <simplelist><member><literal>Number Major Minor RaidDevice State</literal></member><member><literal>0 253 0 0 active sync /dev/dm-0</literal></member><member><literal>1 253 1 1 active sync /dev/dm-1</literal></member><member><literal>2 253 2 2 active sync /dev/dm-2</literal></member>
```
Comment (cjschroder) [inline xml/storage_multipath.xml:3836]: there's them weird member things again :)

File: xml/common_intro_lifecycle_support_i.xml (hunk 1 above)
Comment (lproven) [reply xml/common_intro_lifecycle_support_i.xml:127]: The problem with changing that is that this seems not to be widely observed -- so either this one list will be "correct" or I have the fairly epic job of changing all the bulleted lists in the manual. Or possibly all of the manuals. 

What do you reckon? Leave this instance but bear this in mind for the future, or start changing all of them? There's at least 1 other capitalized bullet-list in this sub-file alone.

File: xml/storage_iscsi.xml (hunk 2 above)
Comment (lproven) [reply xml/storage_iscsi.xml:1307]: Unless I misunderstand, that is what this change does -- its purpose is to insert a nonbreaking space between the digit and the units...

File: xml/storage_iscsi.xml (hunk 3 above)
Comment (lproven) [reply xml/storage_iscsi.xml:1437]: As before, that is what the change was **for**.

File: xml/storage_multipath.xml (hunk 4 above)
Comment (lproven) [reply xml/storage_multipath.xml:66]: OK, reformatted. The groupings make no sense to me; some companies' products are all on a single line, whereas others, notably IBM's, get 1 line per product.

Do you think I should re-group these into a smaller number of lines, or indeed, into a minimal number of lines, or is it best left alone?

File: xml/storage_multipath.xml (hunk 5 above)
Comment (lproven) [reply xml/storage_multipath.xml:112]: I blame the daps-xmlformat tool for this. #grump

Comment (lproven) [reply xml/storage_multipath.xml:112]: Fixed.

File: xml/storage_multipath.xml (hunk 6 above)
Comment (lproven) [reply xml/storage_multipath.xml:291]: `initrd` is not a settings file, though; it is a RAMdisk containing lots of files, supplied to the kernel.

File: xml/storage_multipath.xml (hunk 7 above)
Comment (lproven) [reply xml/storage_multipath.xml:1736]: OK. Fixed.

File: xml/storage_iscsi.xml (hunk 2 above)
Comment (cjschroder) [reply xml/storage_iscsi.xml:1307]: oh yes, it does :)

File: xml/storage_iscsi.xml (hunk 3 above)
Comment (cjschroder) [reply xml/storage_iscsi.xml:1437]: I blame society.

File: xml/storage_multipath.xml [hunk 9]
```diff
@@ -2676,97 +2678,94 @@ prio_args ""</screen>
...
+        <literal>io_num</literal> is the number of read IOs sent to the current
+        path continuously, which are used to calculate the average path
+        latency. Valid values are integers from 2 to 200.
+       </para>
+       <para>
+        <literal>base_num</literal> is the logarithmic base number, used to
+        partition different priority ranks. Valid values are integers from 2 to
+        10. The maximum average latency value is 100s, the minimum is
+        1&nbsp;&mu;s. For example, if <literal>base_num=10</literal>, the paths
+        will begrouped in priority groups with path latency &lt;=1&nbsp;&mu;s,
```
Comment (cjschroder) [inline xml/storage_multipath.xml:2760]: 'be grouped'

File: xml/storage_multipath.xml (hunk 6 above)
Comment (cjschroder) [reply xml/storage_multipath.xml:367]: right, sorry.

File: xml/common_intro_lifecycle_support_i.xml (hunk 1 above)
Comment (cjschroder) [reply xml/common_intro_lifecycle_support_i.xml:127]: IMO stay focused on the task at hand and don't worry about global changes.

File: xml/storage_multipath.xml (hunk 4 above)
Comment (cjschroder) [reply xml/storage_multipath.xml]: That looks way better \o/ The groupings are inconsistent, but IMO as long as the file is reasonably human-readable it's good enough. We have enough to do without going nuts over small details.

File: xml/common_intro_lifecycle_support_i.xml (hunk 1 above)
Comment (lproven) [reply xml/common_intro_lifecycle_support_i.xml:127]: Fixed.

**Comment by cjschroder** [comment]
Stop looking at this before it consumes you :) I pronounce it fit. :+1:

File: xml/common_intro_lifecycle_support_i.xml (hunk 1 above)
Comment (taroth21) [reply xml/common_intro_lifecycle_support_i.xml]: Thanks! For this new snippet in our prefaces, I copied an existing text from a bug IIRC and did not realize that the listitems did not match our style guide rules, my bad, sorry. I'll also fix this in the doc-kit repo which contains the boilerplate texts, otherwise the changes might be overwritten with an upddate from doc-kit in the future.

File: xml/storage_iscsi.xml [hunk 10]
```diff
@@ -1203,7 +1203,7 @@ node.session.iscsi.ImmediateData = Yes
  </sect1>
 
  <sect1 xml:id="sec-iscsi-targetcli">
-  <title>Setting up Software Targets Using targetcli-fb</title>
+  <title>Setting Up Software Targets Using targetcli-fb</title>
```
Comment (sknorr) [inline xml/storage_iscsi.xml:1206]: This makes sense to me (the pronoun belongs to the verb).

File: xml/common_intro_lifecycle_support_i.xml [hunk 11]
```diff
@@ -124,28 +124,28 @@
...
     <para>
-     Some packages shipped as part of the module <emphasis>Workstation
+     some packages shipped as part of the module <emphasis>Workstation
      Extension</emphasis> are L2-supported only.
     </para>
    </listitem>
    <listitem>
     <para>
-     Packages with names ending in <package>-devel</package> (containing header
+     packages with names ending in <package>-devel</package> (containing header
```
Comment (sknorr) [inline xml/common_intro_lifecycle_support_i.xml:148]: These make no sense to me. If there needs to be any punctuation at the end, I'd go for a comma, except on the last bullet. But I'd rather keep these as they were.

File: xml/storage_iscsi.xml [hunk 12]
```diff
@@ -369,8 +369,8 @@
     <title>Partitions</title>
     <para>
      Before you begin, choose the partitions you wish to use for back-end
-     storage. The partitions do not have to be formatted: the iSCSI client can
-     format them when connected, overwriting all existing formatting.
+     storage. The partitions do not have to be formatted&mdash;the iSCSI client
+     can format them when connected, overwriting all existing formatting.
```
Comment (sknorr) [inline xml/storage_iscsi.xml:373]: This change makes no difference, imo. potato/tomahto.

File: xml/storage_iscsi.xml [hunk 13]
```diff
@@ -1298,13 +1298,13 @@ o- / ............................ [...]
     </listitem>
    </itemizedlist>
    <para>
-    To familiarize yourself with functionality of targetcli, set up a local
+    To familiarize yourself with the functionality of targetcli, set up a local
```
Comment (sknorr) [inline xml/storage_iscsi.xml:1301]: Good change.

File: xml/storage_iscsi.xml [hunk 14]
```diff
@@ -1298,13 +1298,13 @@ o- / ............................ [...]
...
     image file as a software target using the <command>create</command> command:
    </para>
 <screen>/backstores/fileio create test-disc <replaceable>/alt</replaceable>/test.img 1G</screen>
    <para>
-    This creates the 1GB <filename>test.img</filename> image in the specified
-    location (in this case <filename>/alt</filename>). Run
+    This creates a 1&nbsp;GB <filename>test.img</filename> image in the
```
Comment (sknorr) [inline xml/storage_iscsi.xml:1306]: Good change

File: xml/storage_iscsi.xml [hunk 15]
```diff
@@ -1324,10 +1324,10 @@ o- / ........................................................... [...]
   o- xen-pvscsi ......................................... [Targets: 0]
 /></screen>   
    <para>
-    The output indicates that there is now a file-based back-store, under the
+    The output indicates that there is now a file-based backstore, under the
```
Comment (sknorr) [inline xml/storage_iscsi.xml:1327]: Merriam-Webster doesn't know the word, it's not defined in our terminology, and I've personally never heard it before. I think it might be a rather unusual compound and would stick to either keeping the `-` or ` ` for this one. (Same for the multiple mentions below.

File: xml/storage_iscsi.xml [hunk 16]
```diff
@@ -1401,7 +1401,7 @@ o- / ............................................................... [...]
     <literal>iqn.2003-01.org.linux-iscsi.e83.x8664:sn.8b35d04dd456</literal>
    </para>
    <para>
-    Note that targetcli has also created and enabled the default target portal
+    Note that <command>targetcli</command> has also created and enabled the default target portal
```
Comment (sknorr) [inline xml/storage_iscsi.xml:1404]: Good change

File: xml/storage_iscsi.xml [hunk 17]
```diff
@@ -1413,11 +1413,11 @@ o- / ............................................................... [...]
     can access the configured target.
    </para>
    <para>
-    Next step is to create a LUN (Logical Unit Number) for the iSCSI
-    target. The best way to do this is to let targetcli assign its name and
+    The next step is to create a LUN (Logical Unit Number) for the iSCSI
+    target. The best way to do this is to let <command>targetcli</command> assign its name and
```
Comment (sknorr) [inline xml/storage_iscsi.xml:1417]: Good changes.

File: xml/storage_iscsi.xml [hunk 18]
```diff
@@ -1371,7 +1371,7 @@ o- / ........................................................... [...]
    </para>
 <screen>/> iscsi/ create</screen>
    <para>
-    Run the <command>ls</command> again
+    Run the <command>ls</command> command again:
```
Comment (sknorr) [inline xml/storage_iscsi.xml:1374]: Good change.

File: xml/storage_iscsi.xml [hunk 19]
```diff
@@ -1482,8 +1482,8 @@ Configuration restored from /etc/target/example.json
     <para>
      To test whether the configured target is working, connect to it using the
      <package>open-iscsi</package> iSCSI initiator installed on the same
-     system (replace <replaceable>HOSTNAME</replaceable> with hostname of the
-     local machine):
+     system (replace <replaceable>HOSTNAME</replaceable> with the hostname of
+     the local machine):
```
Comment (sknorr) [inline xml/storage_iscsi.xml:1486]: Good change.

File: xml/storage_lvm.xml [hunk 20]
```diff
@@ -1647,9 +1647,10 @@ sudo lvreduce /dev/LOCAL/DATA</screen>
     <para>
      To check the exact behavior, run the <command>lvmconfig -l | grep
      global_filter_compat</command> command. If the result shows that the
-     <literal>global_filter_compat</literal> option exists and set to
+     <literal>global_filter_compat</literal> option exists and is set to
      <literal>0</literal>, the behavior described above applies. If the option
-     does not exist or set to <literal>1</literal>, the behavior remains unchanged.
+     does not exist or is set to <literal>1</literal>, the behavior remains
+     unchanged.
```
Comment (sknorr) [inline xml/storage_lvm.xml:1653]: Both the `set` v/ `is set` changes are potato/tomahto changes again. No idea what they're good for.

File: xml/storage_multipath.xml [hunk 21]
```diff
@@ -364,7 +364,8 @@
     that the <filename>initrd</filename> and the installed system behave the
     same regarding the root file system and all other file systems required to
     boot the system. If Multipath is enabled in the system, it also needs to be
-    enabled in the <filename>initrd</filename> and vice versa. See <xref
+    enabled in the <filename>initrd</filename> and vice versa. See
+    <xref
```
Comment (sknorr) [inline xml/storage_multipath.xml:368]: I personally find tags with their attributes on the next line really ugly. Especially like here, where the tag itself is now on an own line.

File: xml/storage_multipath.xml [hunk 22]
```diff
@@ -2676,97 +2678,94 @@ prio_args ""</screen>
...
-             </para>
-        </listitem>
-    </varlistentry>
-</variablelist>
-  
+     These are the arguments for the prioritizer programs that require
+     arguments. Most <literal>prio</literal> programs do not need arguments.
+     There is no default. The values depend on the <literal>prio</literal>
+     setting and whether the prioritizer requires any of the following
+     arguments:
```
Comment (sknorr) [inline xml/storage_multipath.xml:2688]: I hope this block is only an outdenting change, because it's hard to compare the before/after here. (Ideally, reformatting changes should be in a different commit to make them easier to see.)

File: xml/storage_nvmeof.xml [hunk 23]
```diff
@@ -350,13 +350,13 @@ Parameter traddr is now '10.0.0.1'.</screen>
...
-    A resulting <literal>1</literal> suggests &nvme; is enabled, a
+    A resulting <literal>1</literal> means &nvme; is enabled, a
     <literal>0</literal> indicates it is disabled.
    </para>
 
    <para>
-    Next, ensure that the Marvell adapter firmware has at least version
-    8.08.204 by checking the output of the following commend:
+    Next, ensure that the Marvell adapter firmware is at least version
+    8.08.204 by checking the output of the following command:
```
Comment (sknorr) [inline xml/storage_nvmeof.xml:359]: Good changes.

File: xml/storage_raid-leds.xml [hunk 24]
```diff
@@ -821,7 +821,7 @@
    </para>
 <screen>&prompt.sudo;ledctl off={ /dev/sda /dev/sdb }</screen>
    <para>
-    To locate three block devices run one of the following commends (both are
+    To locate three block devices, run one of the following commands (both are
```
Comment (sknorr) [inline xml/storage_raid-leds.xml:824]: Good change.
